src/Process/Pool/Logger/Formatter.php

```diff
@@ -27,6 +27,20 @@ public function getFormattedMessage(MessageInterface $message) : string
         }
     }
 
+    public function getFormattedThrowableMessage(\Throwable $throwable): string
```

Comment:
Tthis is great, but I wonder if having this stateless method on this object is solved when you create the notification API.

I'd like to see the formatting be encapsulated in that notification subsystem with appropriately normalized actors, instead of invoking it like
```php
$message = $this->_getLogger()->getLogFormatter()->getFormattedThrowableMessage($throwable);
```
for all the OO reasons. something like 
```php
$this->getNotifcationService()->setThrowable($thowable);
$this->getNotifcationService()->setLevelCritical();
$this->getNotifcationService()->notify();
```
would be ideal but I'm not blocking this PR waiting for that. Let's just be mindful about what we link to this method because a very soon future us will have to unlink it all.

Reply:
this is probably even closer to the truth
```php
$notification = $this->getNotifcationRepository()->createBuilder()->build();
$notification->setThrowable($thowable);
$notification->setLevelCritical();
$notification->notify();
```

Reply:
I added these examples to the ticket for the notification API. Using our notification API to construct the `level`, `message` and `context` arrays for Monolog's `addRecord()` is up next on my list.